libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -469,7 +469,8 @@ mkQueryPoolStateResult ::
 mkQueryPoolStateResult f ps =
   QueryPoolStateResult
     { qpsrStakePoolParams = Map.mapWithKey stakePoolStateToStakePoolParams restrictedStakePools
-    , qpsrFutureStakePoolParams = Map.mapWithKey stakePoolStateToStakePoolParams restrictedStakePools
+    , qpsrFutureStakePoolParams =
+        Map.mapWithKey stakePoolStateToStakePoolParams (f $ psFutureStakePoolParams ps)
```

Comment:
You're living in the past! 😆 

```suggestion
        Map.mapWithKey stakePoolStateToStakePoolParams (f $ psFutureStakePools ps)
```

Reply:
I went with stupid HLS suggestion :facepalm: